ili2gpkg/test/java/ch/ehi/ili2gpkg/RtreeSpatialIndexTest.java

```diff
@@ -0,0 +1,293 @@
+package ch.ehi.ili2gpkg;
+
+
+import ch.ehi.ili2db.base.Ili2db;
+import ch.ehi.ili2db.gui.Config;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import java.io.File;
+import java.sql.*;
+import java.util.ArrayList;
+
+public class RtreeSpatialIndexTest {
+    private static final String TEST_OUT="test/data/RtreeSpatialIndex/";
+    private static final String FILENAME_XTF=TEST_OUT+"RtreeSpatialIndex.xtf";
+    private static final String MODEL_XTF="RtreeSpatialIndex";
+    private static final String FILENAME_XTF_GPKG=TEST_OUT+"RtreeSpatialIndex.gpkg";
```

Comment:
Diese Datei wird erzeugt oder (Falls ja: Konstante umbennen in FILENAME_GPKG_OUT) ?

Reply:
Genau, die Datei wird basierend auf dem Inhalt der XTF-Datei erstellt.

Ich habe das entsprechend umbenannt